src/PageLayout/interaction.stories.tsx

```diff
@@ -389,5 +390,41 @@ NonStickyPane.play = async ({canvasElement}: {canvasElement: HTMLElement}) => {
   const paragraphRect = paragraph0.getBoundingClientRect()
   expect(isInViewPort(paragraphRect)).toBe(false)
 }
+
+CustomStickyHeader.argTypes = {
+  sticky: {
+    type: 'boolean',
+    defaultValue: true
+  },
+  stickyTop: {
+    type: 'string',
+    defaultValue: '8rem'
+  },
+  numParagraphsInPane: {
+    type: 'number',
+    defaultValue: 10
+  },
+  numParagraphsInContent: {
+    type: 'number',
+    defaultValue: 30
+  }
+}
+
+CustomStickyHeader.play = async ({canvasElement}: {canvasElement: HTMLElement}) => {
+  const canvas = within(canvasElement)
+  const contentToScroll = await canvas.getByTestId('content3')
+  contentToScroll.scrollIntoView()
+
+  // fireEvent alternative?
+  const storyWindow = await canvas.getByTestId('story-window')
+  await fireEvent.scroll(storyWindow, {top: 600})
```

Comment:
I wanted to give it a go to more descriptive event names, but can't seem to make this work :( See the [doc](https://testing-library.com/docs/dom-testing-library/api-events) Maybe I am missing something? @pksjce

Reply:
They don't yet have a descriptive name feature. However, storybook 7.0.0 is going to support sections within interaction tests I believe!

Reply:
Ahh cool! I'll remove this then, and we can try later when available.